css/off.canvas.drawer.css

```diff
@@ -0,0 +1,241 @@
+#drawer-button {
+  float:right;
+  color:#000;
+  border:none;
+  background-color:rgba(0, 0, 0, 0);
+  font-family: Arial,Sans-Serif;
+  padding:10px;
+  font-size:14px;
+  line-height: 1.75em!important;
```

Comment:
It may be good if we don't have to use `!important`, It's an id so it's pretty strong in specificity and harder to override in a child theme.